File: C#/Practice/LeetCode/Easy/Count the Number of Consistent Strings.cs
// https://leetcode.com/problems/count-the-number-of-consistent-strings/description/

public class Solution {
    public int CountConsistentStrings(string allowed, string[] words) {
        int consistentStringsCounter = 0;
        for (int i = 0; i < words.Length; i++) {
            for (int j = 0; j < words[i].Length; j++) {
                if (!allowed.Contains(words[i][j])) {
                    break;
                }
                if (j == words[i].Length - 1) {
                    consistentStringsCounter++;
                }
            }
        }
        return consistentStringsCounter;        
    }
}

File: TypeScript/Practice/LeetCode/Easy/Count the Number of Consistent Strings.ts
// https://leetcode.com/problems/count-the-number-of-consistent-strings/description/

function countConsistentStrings(allowed: string, words: string[]): number {
    let consistentStringsCounter: number = 0;
    for (let i = 0; i < words.length; i++) {
        for (let j = 0; j < words[i].length; j++) {
            if (!allowed.includes(words[i][j])) {
                break;
            }
            if (j === words[i]. length - 1) {
                consistentStringsCounter++;
            }
        }
    }
    return consistentStringsCounter;
};